tests/unit_tests/blas/include/reference_blas_templates.hpp

```diff
@@ -39,74 +39,2377 @@
 #endif
 
 extern "C" {
-static LIB_TYPE h = NULL;
-static void (*csrot_p)(const int *N, void *X, const int *incX, void *Y, const int *incY,
+static LIB_TYPE h_libblas = NULL;
+static LIB_TYPE blas_library() {
+    if (h_libblas == NULL) {
+        h_libblas = GET_LIB_HANDLE(REF_BLAS_LIBNAME);
+        if (h_libblas == NULL) {
+            throw oneapi::mkl::library_not_found("BLAS", "blas_library()",
```

Comment:
Does it make sense to print REF_BLAS_LIBNAME here to provide more information about what exact library we tried to load?

Reply:
Addressed in the commit [f17ce6c](https://github.com/oneapi-src/oneMKL/pull/251/commits/f17ce6c48dc27c299a4a3f8dc4842e8d2d5a604b).

An example of the error message:
```
C++ exception with description "oneMKL: BLAS/cblas_library(): library not found: failed to load CBLAS library /nfs/site/proj/mkl/mirror/NN/tools/netlib/3.10.0_5d4180c/blas/lnx/lib/libcblas.so" thrown in the test body.
```